nifi-nar-bundles/nifi-update-attribute-bundle/nifi-update-attribute-processor/src/main/java/org/apache/nifi/processors/attributes/UpdateAttribute.java

```diff
@@ -144,6 +149,21 @@ public ValidationResult validate(String subject, String input, ValidationContext
         }
     };
 
+    public static final AllowableValue FAIL_STOP = new AllowableValue("stop", "Penalize", "Penalize FlowFiles." +
```

Comment:
Hey @MikeThomsen,
This was a little confusing when I did a build and was configuring the processor.  What do you think of making this penalize instead of stop.  At first I thought I was missing an option in the drop down.  While the value is supposed to be dev facing and the display property is user facing, the tooltips in the UI make use of the value and not the display property.

Reply:
I'll check it out and do a separate ticket if needed.

Reply:
@apiri @zenfenan FYI, I just manually pushed that commit. It was a one liner on an approved commit, so didn't feel like it needed a review.